components/selection/demo/selection.html

```diff
@@ -56,22 +57,19 @@ <h2>Multiple Selection</h2>
 			<d2l-demo-snippet>
 				<template>
 					<d2l-demo-selection>
-						<div class="d2l-selection-header">
-							<div style="flex: auto;">
-								<d2l-selection-select-all></d2l-selection-select-all>
-								<d2l-selection-action text="Bookmark" icon="tier1:bookmark-hollow" requires-selection></d2l-selection-action>
-								<d2l-selection-action text="Settings" icon="tier1:gear"></d2l-selection-action>
-								<d2l-selection-action-dropdown text="Actions 1" requires-selection>
-									<d2l-dropdown-menu>
-										<d2l-menu label="Actions">
-											<d2l-selection-action-menu-item text="Action 1"></d2l-selection-action-menu-item>
-											<d2l-selection-action-menu-item text="Action 2"></d2l-selection-action-menu-item>
-										</d2l-menu>
-									</d2l-dropdown-menu>
-								</d2l-selection-action-dropdown>
-							</div>
-							<d2l-selection-summary style="flex: none;"></d2l-selection-summary>
-						</div>
+						<d2l-selection-header no-sticky>
+							<d2l-selection-action text="Bookmark" icon="tier1:bookmark-hollow" requires-selection></d2l-selection-action>
+							<d2l-selection-action text="Settings" icon="tier1:gear"></d2l-selection-action>
+							<d2l-selection-action-dropdown text="Actions 1" requires-selection>
+								<d2l-dropdown-menu>
+									<d2l-menu label="Actions">
+										<d2l-selection-action-menu-item text="Action 1"></d2l-selection-action-menu-item>
+										<d2l-selection-action-menu-item text="Action 2"></d2l-selection-action-menu-item>
+									</d2l-menu>
+								</d2l-dropdown-menu>
+							</d2l-selection-action-dropdown>
+						</d2l-selection-header>
```

Comment:
These selection demos (and the ones in the readme) all had a custom-made selection header, so I've replaced those all now with `d2l-selection-header`. I've used the `no-sticky` attribute to keep the behaviour/appearance the same as before.

Reply:
The previous demo was more just set up to show the various building blocks, so now that we have a new, somewhat higher level building block, I'd probably just render it with its default sticky mode, or provide two demos... one sticky and one not.